### PR TITLE
[fetcheus]  add fetch head box collision

### PR DESCRIPTION
**Pull request by sktometometo**
- add fetch headbox collision object when initializing fetch-interface

Please merge this pullreq after mergining https://github.com/jsk-ros-pkg/jsk_robot/pull/1174

### CODE REVIEW

**Comment by 708yamaguchi** [comment]
Regarding to https://github.com/jsk-ros-pkg/jsk_robot/pull/1186/commits/83295bacfb9798bc338be11f7d7f7cb75dc93d64#diff-d7eb7619176f5641c713e05e846088d4R203-R205

You can use `move-coords` method to adapt any posture of fetch.
For example,
```
(setq *fetch-headbox* (make-cube 100 201 120))
(send *fetch-headbox* :move-coords (send *fetch* :head_pan_link_lk :worldcoords) (send *fetch* :base_link_lk :worldcoords))
(send *fetch-headbox* :translate #f(33.75 0 150) (send *fetch* :head_pan_link_lk :worldcoords))
```
(You may use slot variable `robot` instead of `*fetch*`.)

**Comment by sktometometo** [comment]
@708yamaguchi 
I have fixed :add-headbox-collision-objecct method according to your advice.
Thank you.

File: jsk_fetch_robot/fetcheus/fetch-interface.l [hunk 1]
```diff
@@ -191,7 +198,38 @@ Example: (send self :gripper :position) => 0.00"
...
+  (:delete-keepout-collision-object ()
+    (send *co* :delete-object-by-id "keepout"))
+  (:add-ground-collision-object ()
+    (let ((cube (make-cube 200 350 50))
+          (ground (make-cylinder 1000 50)))
+      (send cube :translate #f(-120 0 0))
+      (send ground :translate #f(0 0 -25))
+      (setq ground (body- ground cube))
+      (send ground :translate #f(0 0 -25))
+      (send *co* :add-object ground :frame-id base-frame-id :object-id "ground")))
```
Comment (k-okada) [inline jsk_fetch_robot/fetcheus/fetch-interface.l]: if https://github.com/jsk-ros-pkg/jsk_pr2eus/pull/423/files is correct, this commit is wrong.

Comment (knorth55) [reply jsk_fetch_robot/fetcheus/fetch-interface.l]: this is correct because this pr uses `:frame-id`, not`:frame_id`.

**Comment by knorth55** [comment]
@sktometometo is this still wip?

**Comment by sktometometo** [comment]
@knorth55 I think it may be not confirmed by experiment that this change does work well.

**Comment by knorth55** [comment]
please check this PR if it is working properly

**Comment by knorth55** [comment]
@sktometometo can you also add the latest urdf and meshes?
cc. @708yamaguchi

**Review by knorth55 — APPROVED** [review]

**Comment by k-okada** [comment]
please update labels...

**Pull request merged**